scripts/import_proxy_certs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Nits:
- Maybe make this the first commit in the PR, as it fixes something that was already wrong before.
- If referring to more than one file, consider using the parent directory as prefix in the title instead of list all files: `scripts: ...`.

Reply:
I was referring to simply reordering the commits in this PR, making the Shebang fix the first in the history of this branch, but that was completely optional anyway.